libs/agno/agno/document/reader/csv_reader.py

```diff
@@ -6,6 +7,8 @@
 from typing import IO, Any, List, Union
 from urllib.parse import urlparse
 
+import aiofiles
```

Comment:
This would be a special installation requirement. I don't think it is built in? So you'll have to add to pyproject.toml and have a import check here

Reply:
done